src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -176,6 +176,14 @@ public static string ParameterCompoundType(int index)
          //end of  delete
       }
 
+      public static class RegistryPaths
+      {
+         public const string PKSIM_REG_PATH = @"Open Systems Pharmacology\PK-Sim\";
+         public const string MOBI_REG_PATH = @"Open Systems Pharmacology\MoBi\";
+         public const string INSTALL_DIR = "InstallDir";
+         public const string INSTALL_PATH = "InstallPath";
```

Comment:
This is actually the constant that you want to read in the registry. This gives you the PATH where PK-Sim is installed